priv/blogs/2022/07-29-observing-elixir-with-lightstep.md

```diff
@@ -0,0 +1,222 @@
+# Observability in Elixir with OpenTelemetry, Lightstep, and Honeycomb
+%{
+    title: "Observability in Elixir with OpenTelemetry, Lightstep, and Honeycomb",
+    author: "Dave Lucia",
+    tags: ~w(elixir observability lightstep),
+    description: "How to integrate your Elixir application with Open Telemetry to make it observable in Lightstep and Honeycomb's reliability platform"
+}
+---
+## What happens when your server goes bump in the night?
+As an [Elixir developer](https://elixir-lang.org/), I get more sleep than most when an issue occurs at 2AM.  In my Elixir app, [supervisors](https://hexdocs.pm/elixir/1.13/Supervisor.html) restart processes when errors occur, keeping my systems available through dropped database connections, exceptions, etc.
+
+When I wake up in the morning, its time to figure out what went wrong, fix the bug, and get back to feature development. *Finding the root-cause of issues is hard*, especially when the issue is not a simple amendment to an [if/else/case expression](https://hexdocs.pm/elixir/1.13/Kernel.SpecialForms.html#case/2), but a performance regression in your infrastructure or database. In order to get to the bottom of these problems, we must have a way to understand our running system. We can achieve this by practicing observability and putting the right tools in place. 
+
+## What is observability?
+
+Observability is simply the ability to observe your running system, with the purpose of understanding its behavior. Forget the [three pillars](https://youtu.be/EJV_CgiqlOE), or anything else you might have read, observability is all about information, tools, and analysis to get to the bottom of your burning questions faster.
+
+If you've ever had questions like:
+
+* Why does my `/api/foo/thing` route suddenly have a P50 of triple its normal latency?
```

Comment:
```suggestion
* Why does my `/api/foo/thing` route's latency suddenly have triple the normal average?
```

This isn't an exact translation because p50 is the median, but that also isn't what people normally know